client/app/react-components/QueryEditor.js

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { map } from 'lodash';
+
+import { OverlayTrigger, Tooltip } from 'react-bootstrap';
```

Comment:
We're adopting [Ant](https://ant.design/docs/react/introduce) as UI library (already used for date parameters). So in this case we should use [Ant's Tooltip component](https://ant.design/components/tooltip/).

Checkout the data parameters code on how to include its stylesheets.